src/pages/docs.py

```diff
@@ -13,37 +13,33 @@
     - [Purpose of the Dashboard](#purpose-of-the-dashboard)
     - [Who Should Use This Dashboard?](#who-should-use-this-dashboard)
 
-    ### **2. Key Concepts**
-    - [Glossary](#glossary)
-        - [Basic Concepts](#basic-concepts)
-        - [Inspect AI Concepts](#inspect-ai-concepts)
-
-    ### **3. Metrics and Scoring**
+    ### **2. Glossary**
+    - [Basic Concepts](#basic-concepts)
+    - [Inspect AI Concepts](#inspect-ai-concepts)
     - [Performance Metrics](#performance-metrics)
     - [Statistical Analysis](#statistical-analysis)
-    - [Score Interpretation](#score-interpretation)
 
-    ### **4. Implementation Details**
+    ### **3. Implementation Details**
     - [Evaluation Pipeline](#evaluation-pipeline)
     - [Model Metadata](#model-metadata)
 
-    ### **5. Visualization and Interpretation**
+    ### **4. Visualization and Interpretation**
     - [Understanding the Dashboard](#understanding-the-dashboard)
         - [Pairwise Analysis](#pairwise-analysis)
             - [Significance-Based Color Coding](#significance-based-color-coding)
+    - [Score Interpretation](#score-interpretation)
+
+    ### **5. Data Access & Contribution**
+    - [How to Download and Use the Data](#how-to-download-and-use-the-data)
+    - [How to Contribute to the Dashboard](#how-to-contribute-to-the-dashboard)
+    - [How to Cite the Dashboard](#how-to-cite-the-dashboard)
 
     ### **6. Frequently Asked Questions (FAQ)**
     - [How do I interpret confidence intervals?](#how-do-i-interpret-confidence-intervals)
     - [What if a model's score is not statistically significant?](#what-if-a-models-score-is-not-statistically-significant)
     - [How are models selected for evaluation?](#how-are-models-selected-for-evaluation)
     - [How to reproduce the results?](#how-to-reproduce-the-results)
 
-    ### **7. Data Access & Contribution**
-    - [How to Download and Use the Data](#how-to-download-and-use-the-data)
-    - [How Did We Calculate the Number of Epochs?](#how-did-we-calculate-the-number-of-epochs)
```

Comment:
this one didn't have an associated explanation yet, not sure if that is a TODO item